src/Integration.UnitTests/Helpers/AbsoluteFilePathLocatorTests.cs

```diff
@@ -140,12 +151,15 @@ public void Locate_RelativePathHasForwardSlashes_PathFixedToBackSlashes()
             vsUiShell.VerifyNoOtherCalls();
         }
 
-        private IServiceProvider CreateServiceProvider(IVsUIShellOpenDocument vsUiShellOpenDocument)
+        private IVsUIServiceOperation CreateServiceOperation(IVsUIShellOpenDocument svcToPassToCallback)
```

Comment:
Since the old class didn't handle threading at all, fixing up the tests is simple - just need to swap `IServiceProvider` for `IVsUIServiceOperation`.